google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractReadContext.java

```diff
@@ -348,6 +356,7 @@ void initTransaction() {
         if (transactionId != null) {
           return;
         }
+        OpenTelemetryTraceUtil.addEvent(openTelemetrySpan, "Creating Transaction");
         span.addAnnotation("Creating Transaction");
```

Comment:
Would it make sense to create a util method somewhere that takes both an OpenTelemetry span and an OC span as an argument, and then add an event/annotation to both? That way we know that we always use the same event name, and we can keep this on one line.

Reply:
Done